### DISCUSSION TITLE
How do I hide/disable some of the social login options that I don't want to use for authentication?

### DISCUSSION THREAD

**Discussion by shahbaz17** [Web3Auth/Web3Auth#488] [Help]
For `@web3auth/modal` package.

**Comment by shahbaz17** [comment]
```js
await web3auth.initModal({
  modalConfig: {
    [WALLET_ADAPTERS.OPENLOGIN]: {
      label: "openlogin",
      loginMethods: {
        google: {
          name: "google login",
          logoDark: "url to your custom logo which will shown in dark mode",
        },
        facebook: {
          // it will hide the facebook option from the Web3Auth modal.
          showOnModal: false,
        },
      },
      // setting it to false will hide all social login methods from modal.
      showOnModal: true,
    },
  },
});
```

https://web3auth.io/docs/sdk/web/web3auth/initialize#example-4

**Reply by jackbl1** [reply]
Just a quick note that there may have been some update in the interface such that under 'facebook' you will now need both a 'showOnModal' and a 'name' field or else this will not work. So it will now need to look like:

await web3auth.initModal({
  modalConfig: {
    [WALLET_ADAPTERS.OPENLOGIN]: {
      label: "openlogin",
      loginMethods: {
        google: {
          name: "google login",
          logoDark: "url to your custom logo which will shown in dark mode",
        },
        facebook: {
          // it will hide the facebook option from the Web3Auth modal.
          name: '',
          showOnModal: false,
        },
      },
      // setting it to false will hide all social login methods from modal.
      showOnModal: true,
    },
  },
});
